nucypher/cli/literature.py

```diff
@@ -311,6 +311,14 @@
 
 NO_FEE_TO_WITHDRAW = "No policy fee can be withdrawn."
 
+TOKEN_REWARD_CURRENT = 'Available staking rewards: {reward_amount:.18g} NU.'
+
+TOKEN_REWARD_PAST_HEADER = 'Staking rewards in the last {periods} periods ({days} days):'
+
+TOKEN_REWARD_PAST = 'Total staking rewards: {reward_amount:.18g} NU.'
```

Comment:
Rather than using floats, for dealing with NU tokens it's better to resort to the `NU` class is `nucypher.blockchain.eth.token`, which allows you to easily convert between NU and NuNits. It implements `__round__`, so you can say something like `round(amount, 2)` to get up to 2 decimal places.